homeassistant/components/uhooair/coordinator.py

```diff
@@ -0,0 +1,39 @@
+"""Imports for coordinator.py."""
+
+import asyncio
+
+from uhooapi import Client, Device
+
+from homeassistant import core
+from homeassistant.helpers.update_coordinator import DataUpdateCoordinator, UpdateFailed
+
+from .const import DOMAIN, LOGGER, UPDATE_INTERVAL
+
+
+class UhooDataUpdateCoordinator(DataUpdateCoordinator):
```

Comment:
```suggestion
class UhooDataUpdateCoordinator(DataUpdateCoordinator[..]):
```
Add the return type of `_async_update_data`